docs/guides/included-middleware.md

```diff
@@ -0,0 +1,41 @@
+Crow contains some middlewares that are ready to be used in your application.
+
+Make sure you understand how to enable and use [middleware](../middleware/).
+
+## Cookies
+Include: `crow/middlewares/cookie_parser.h` <br>
+Examples: `examples/middlewars/example_cookies.cpp`
+
+This middleware allows to read and write cookies by using `CookieParser`. Once enabled, it parses all incoming cookies.
+
+Cookies can be read and written with the middleware context. All cookie attributes can be changed as well.
+
+```c++
```

Comment:
it might be best to use `cpp` instead of `c++` since `cpp` is a more recognized abbreviation (works with both github and mkdocs)